test/test_data/multipart_string.py

```diff
@@ -0,0 +1 @@
+"string1" "string2" "string3"
```

Comment:
These should be included in the newly created `TEST_SOURCES` array from #134.

Reply:
Thanks! I rebased to pick up the latest changes and forgot about moving these